lib/mpl_toolkits/mplot3d/proj3d.py

```diff
@@ -146,6 +144,29 @@ def proj_transform(xs, ys, zs, M):
 transform = proj_transform
 
 
+def proj_transform_vectors(vecs, M):
```

Comment:
please make this private (ie underscore-prefixed).  we're trying to make all of proj3d private, basically, so that we are not bound by backcompat concerns when trying to optimize it.

Reply:
Makes sense, will do!